camera_calibration_parsers/src/convert.cpp

```diff
@@ -33,30 +33,35 @@
 *********************************************************************/
 
 #include "camera_calibration_parsers/parse.h"
-#include <ros/console.h>
+
 #include <cstdio>
+#include <string>
+
+#include <rclcpp/logging.hpp>
+#include <sensor_msgs/msg/camera_info.hpp>
 
 using namespace camera_calibration_parsers;
 
-int main(int argc, char** argv)
+int main(int argc, char ** argv)
 {
+  auto logger = rclcpp::get_logger("convert");
```

Comment:
I suspect logger names are global, right? In that case, consider using a more specific name?

Reply:
In this case, this would only be for this executable (yml to ini or ini to yml), so it's probably safe to be generic.